### PR TITLE
Fix file alreadly exists issue in luks

### PR DESCRIPTION
**Pull request by chunfuwen**
Previous case may leave dirty file in ceph server,
add pre clean up before case is exectuing actually.

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
previous error log:

2020-03-31 17:04:55,118 virtual_disks_lu L0392 DEBUG| device source is: rbd:avocado-vt-pool/rbd.img:mon_host=libvirtauthceph.usersys.redhat.com:keyring=/var/tmp/avocado__dmzwoyh/ceph.key
2020-03-31 17:04:55,119 process          L0626 INFO | Running 'qemu-img create -f luks --object secret,id=sec0,data=`printf 'password' | base64`,format=base64 -o key-secret=sec0 rbd:avocado-vt-pool/rbd.img:mon_host=libvirtauthceph.usersys.redhat.com:keyring=/var/tmp/avocado__dmzwoyh/ceph.key 500M'
2020-03-31 17:04:55,387 process          L0458 DEBUG| [stderr] qemu-img: rbd:avocado-vt-pool/rbd.img:mon_host=libvirtauthceph.usersys.redhat.com:keyring=/var/tmp/avocado__dmzwoyh/ceph.key: error rbd create: File exists
2020-03-31 17:04:55,388 process          L0458 DEBUG| [stdout] Formatting 'rbd:avocado-vt-pool/rbd.img:mon_host=libvirtauthceph.usersys.redhat.com:keyring=/var/tmp/avocado__dmzwoyh/ceph.key', fmt=luks size=524288000 key-secret=sec0
2020-03-31 17:04:55,391 process          L0714 INFO | Command 'qemu-img create -f luks --object secret,id=sec0,data=`printf 'password' | base64`,format=base64 -o key-secret=sec0 rbd:avocado-vt-pool/rbd.img:mon_host=libvirtauthceph.usersys.redhat.com:keyring=/var/tmp/avocado__dmzwoyh/ceph.key 500M' finished with 1 after 0.2663097381591797s
2020-03-31 17:04:55,587 virsh            L1434 DEBUG| Undefine VM avocado-vt-vm1

**Comment by chunfuwen** [comment]
new pass log:

Command 'ps -o comm 1' finished with 0 after 0.015066862106323242s
[stdout] systemd
legacy start libvirtd daemon IGNORED with function name: restart
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.luks.negative_test.wrong_password.coldplug.device_disk.ceph_test.enable_auth.auth_in_source.encryption_out_source

Requested MAC address release from persistent vm avocado-vt-vm1. Ignoring.
Test results available in /root/avocado/job-results/job-2020-04-02T04.20-5eb522e
found extension EntryPoint.parse('json = avocado.plugins.jsonresult:JSONResult')
found extension EntryPoint.parse('xunit = avocado.plugins.xunit:XUnitResult')

File: libvirt/tests/src/virtual_disks/virtual_disks_luks.py [hunk 1]
```diff
@@ -49,6 +49,10 @@ def encrypt_dev(device, params):
         """
         password = params.get("luks_encrypt_passwd", "password")
         size = params.get("luks_size", "500M")
+        # Upload to ceph server is time-consuming, keep the size in low value.
+        enable_auth_test = "yes" == params.get("enable_auth")
+        if enable_auth_test:
+            size = "100M"
```
Comment (kylazhang) [inline libvirt/tests/src/virtual_disks/virtual_disks_luks.py:55]: I think it is better to set in cfg

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_luks.py:55]: original cfg don't have any config parameter for size, so I think we don't need introduce new variable in cfg file.

Comment (kylazhang) [reply libvirt/tests/src/virtual_disks/virtual_disks_luks.py:55]: we have in cfg, luks_size = "500M" then in the enable_auth test with luks_size = "100M" will be ok, I saw there are 2 places need to set. And this section of change will not needed

**Pull request merged**